pkgs/development/python-modules/dm-tree/default.nix

```diff
@@ -24,7 +23,11 @@ buildPythonPackage rec {
     hash = "sha256-VvSJTuEYjIz/4TTibSLkbg65YmcYqHImTHOomeorMJc=";
   };
 
-  patches = [ ./cmake.patch ];
+  patches = [
+    ./0001-don-t-rebuild-abseil.patch
+    ./0002-don-t-fetch-pybind11.patch
+    ./0003-do-not-use-apple-if-in-cmake.patch
```

Comment:
let's use fetchpatch here since it sounds like these are available online

also should we guard these patches on `platform = darwin`?

Reply:
Done